transaction/extra/src/signed_contingent_input.rs

```diff
@@ -215,8 +316,10 @@ pub enum SignedContingentInputError {
     MissingPartialFillChange,
     /// Index out of bounds
     IndexOutOfBounds,
-    /// Fractional change amount was zero
+    /// Partial fill change amount was zero
     ZeroPartialFillChange,
+    /// Partial fill output amount was zero
+    ZeroPartialFillOutput,
```

Comment:
this is not currently an error from the consensus network's point of view, but if we change input rules validation to use `U64Ratio`, then it probably will be.

I think that's okay, there's not any interesting use-case for having zero-value partial fill outputs.